ledger-api/grpc-definitions/com/daml/ledger/api/v1/command_service.proto

```diff
@@ -17,6 +17,12 @@ option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 
 // Command Service is able to correlate submitted commands with completion data, identify timeouts, and return contextual
 // information with each tracking result. This supports the implementation of stateless clients.
+// ``INVALID_ARGUMENT`` gRPC error will be returned in case of malformed payloads and missing required fields.
```

Comment:
I think we should either specify everywhere that these codes are gRPC errors or leave that implicit everywhere.

Reply:
I'll leave that as implicit.